.github/workflows/rust.yml

```diff
@@ -258,27 +252,27 @@ jobs:
         # FIXME: we need to update the Makefile to do this correctly
         case ${NAME} in
         'test')
-          cargo llvm-cov nextest --lcov --output-path lcov-${{ matrix.target }}-${{ matrix.special_features }}.info \
+          cargo llvm-cov nextest --lcov --output-path lcov-${{ matrix.target }}.info \
               --partition count:${PARTITION}/4 \
-              --workspace --features "cairo-1-hints,  test_utils, ${{ matrix.special_features }}"
+              --workspace --features "cairo-1-hints,  test_utils"
           ;;
         'test-no_std')
-          cargo llvm-cov nextest --lcov --output-path lcov-${{ matrix.target }}-${{ matrix.special_features }}.info \
+          cargo llvm-cov nextest --lcov --output-path lcov-${{ matrix.target }}.info \
               --partition count:${PARTITION}/4 \
-              --workspace --no-default-features --features "${{ matrix.special_features }}"
+              --workspace --no-default-features
           ;;
         'test-wasm')
           # NOTE: release mode is needed to avoid "too many locals" error
-          wasm-pack test --release --node vm --no-default-features --features "${{ matrix.special_features }}"
+          wasm-pack test --release --node vm --no-default-features
```

Comment:
We should restore this, as we added the "extensive-hints" feature recently and it should be tested